api/src/nv_ingest_api/internal/store/image_upload.py

```diff
@@ -134,35 +106,61 @@ def _upload_images_to_minio(df: pd.DataFrame, params: Dict[str, Any]) -> pd.Data
             image_type = "png"
             if doc_type == ContentTypeEnum.IMAGE:
                 image_metadata = metadata.get("image_metadata", {})
-                image_type = image_metadata.get("image_type", "png")
+                raw_image_type = image_metadata.get("image_type", "png")
+                # Handle both enum and string values
+                if hasattr(raw_image_type, "value"):
+                    # It's an enum, get the string value
+                    image_type = raw_image_type.value.lower()
+                else:
+                    image_type = str(raw_image_type).lower()
+            elif doc_type == ContentTypeEnum.STRUCTURED:
```

Comment:
all of this doc_type logic both here and below, can it be pulled out into a function as well?